addons/hw_drivers/iot_handlers/interfaces/SerialInterface.py

```diff
@@ -10,9 +10,11 @@ class SerialInterface(Interface):
     connection_type = 'serial'
 
     def get_devices(self):
-        serial_devices = {}
-        for port in serial.tools.list_ports.comports():
-            serial_devices[port.device] = {
-                'identifier': port.device
-            }
+        serial_ports = serial.tools.list_ports.comports()
+        serial_devices = {
+            port.device: {'identifier': port.device}
+            for port in serial_ports
+            if port.subsystem == 'use-serial'
```

Comment:
with this change we won't detect blackboxes anymore because for a blackbox the `port.subsystem` is "usb-serial"